composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -43,8 +45,9 @@ def set_batch_sequence_length(batch: Dict[str, torch.Tensor], curr_seq_len: int,
     Args:
         batch (Dict[str, Tensor]): The input batch to the model, must be a dictionary.
         curr_seq_length (int): The desired sequence length to apply.
-        truncate (bool, optional): Truncate sequences early, or reshape tensors to create
-            new examples out of the extra tokens. Default: ``True``.
+        preserve_end_of_sequence (bool, optional): Preserve the end-of-sequence of the batch when
```

Comment:
Do we still want to preserve the `truncate=False` option for backwards compatibility?

Reply:
Good point. I guess we shouldn't assume that others aren't using `truncate=False` just because we aren't. I will add it back in.